src/Raven.Server/Documents/Indexes/IndexDefinitionBase.cs

```diff
@@ -29,6 +29,8 @@ public abstract class IndexDefinitionBase
 
         public IndexState State { get; set; }
 
+        internal long LastStateChangeRaftIndex { get; set; }
```

Comment:
rename this class to IndexDefinitionBaseServerSide. We already have similar convention with other classes like that.